pipx/commands.py

```diff
@@ -180,14 +180,7 @@ def _remove_all_expired_venvs():
 
 
 def _http_get_request(url: str):
-    parts = urllib.parse.urlparse(url)
-    conn = http.client.HTTPSConnection(parts.hostname)
-    conn.request("GET", parts.path)
-    response = conn.getresponse()
-    if response.status != 200:
-        raise PipxError(response.reason)
-
-    return response.read().decode("utf-8")
+    return urllib.request.urlopen(url).read().decode("utf-8")
```

Comment:
Nice! This is much simpler. Are there any downsides?

Also, can the status code be checked (`getcode()`) and an appropriate `PipxError` error be raised if it doesn't start with a `2` or `3`?

Would be nice if we could detect the actual encoding as described in this section of chardet: https://chardet.readthedocs.io/en/latest/faq.html#yippie-screw-the-standards-ill-just-auto-detect-everything